debian/debathena-pyhesiodfs.postinst

```diff
@@ -25,8 +25,17 @@ set -e
 
 case "$1" in
     configure)
+
+    # Adding pyhesiodfs only if the local user group fuse exists.
+    # The group fuse no longer exists in jessie.
+    # If the group fuse is needed, it should have been added when the package 
+    # fuse, a dependency, was configured, before this script is run.
+    if grep "^fuse:" /etc/group > /dev/null 2>&1; then
```

Comment:
Does `grep -q` only suppress `stdout` and not `stderr`? If so, then according to the GNU grep man page, `grep "^fuse:" /etc/group > /dev/null` might match that behavior best.